packages/terra-site/src/examples/date-picker/DatePickerDefault.jsx

```diff
@@ -1,10 +1,28 @@
 import React from 'react';
 import DatePicker from 'terra-date-picker';
 
-const DatePickerExample = () => (
-  <DatePicker
-    name="date-input-default"
-  />
-);
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: '' };
+    this.handleDateChange = this.handleDateChange.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>Selected ISO Date: {this.state.date}</p>
```

Comment:
After thinking about this a little more, perhaps, the handleChange function should strip out the time part in the ISO string before passing it down. Thoughts anyone?